source/client-side-encryption/client-side-encryption.rst

```diff
@@ -708,20 +708,44 @@ ClientEncryption
 
       // Creates a new key document and inserts into the key vault collection.
       // Returns the \_id of the created document as a UUID (BSON binary subtype 4).
-      createKey(kmsProvider: String, opts: Optional<DataKeyOpts>): Binary;
+      createKey(kmsProvider: String, opts: Optional<DataKeyOpts>): UUID;
 
       // An alias function equivalent to createKey.
-      createDataKey(kmsProvider: String, opts: Optional<DataKeyOpts>): Binary;
+      createDataKey(kmsProvider: String, opts: Optional<DataKeyOpts>): UUID;
 
       // Decrypts multiple data keys and (re-)encrypts them with a new masterKey, or with their current masterKey if a new one is not given.
       // Returns a RewrapManyDataKeyResult.
       rewrapManyDataKey(filter: Document, opts: Optional<RewrapManyDataKeyOpts>): RewrapManyDataKeyResult;
 
+      // Removes the key document with the given \_id from the key vault collection.
```

Comment:
Since this is appearing within a `.. code` block, is is necessary to escape `\_id` like that? This also applies to comment blocks above, which predate this particular PR.